optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -429,7 +443,8 @@ def best_params(self) -> Dict[str, Any]:
             params.update(self.lgbm_params)
             return params
 
-    def _parse_args(self, *args: Any, **kwargs: Any) -> None:
+    def _parse_args(self, *args, **kwargs):
+        # type: (Any, Any) -> None
```

Comment:
How come you preferred the comment styled type hint here?

Reply:
Thank you for pointing out. I think I failed the operation of git rebase. Fixed: https://github.com/optuna/optuna/pull/1217/commits/c230464ae16b6ce90643fd62306c67c7a4b1b42a.